src/app/shared/components/text-tooltip/text-tooltip.component.html

```diff
@@ -1 +1,3 @@
-{{ text }}
+<span *ngIf="text">
```

Comment:
You don't need a `span` for this. 
If you need to add an "element" just for conditionals then `ng-container` is your friend, since `ng-container` does not leave an element in the dom.

Reply:
@LindvedKrvang `ng-container` dose not leave element, but if we apply the css to 
```
:host {
  display: flex;
  background: var(--rundown-scrollbar-color);
  color: var(--light-gray-color);
  padding: 4px;
  border-radius: 3px;
  font-size: 0.8rem;
  box-shadow: 0 0 1px var(--dark-gray-color);
  white-space: nowrap;
}
```
The host element is there and we got this dot bug:
![Screenshot 2024-04-09 132337](https://github.com/tv2/ng-sofie/assets/152977713/80ce3e4e-d796-4f76-b43c-a7324497f647)

Reply:
Ahh, then leave it as is :)